vars.tf

```
@@ -0,0 +1,98 @@
+variable "aws_region" {
+  default = "us-east-1"
+}
+
+variable "aws_access_key" {
+}
```

This allows for running terraform on the command-line using environment variable `AWS_ACCESS_KEY_ID` to provide the value.
```suggestion
variable "aws_access_key" {
  default = null
}
```